tests/fixtures-valid.js

```diff
@@ -27,21 +60,41 @@ const uniqueValues = {
 };
 
 const promises = [];
-
-// search fixture files
 const fixturePath = path.join(__dirname, `..`, `fixtures`);
-for (const manKey of fs.readdirSync(fixturePath)) {
-  const manDir = path.join(fixturePath, manKey);
-
-  // files in manufacturer directory
-  if (fs.statSync(manDir).isDirectory()) {
-    for (const file of fs.readdirSync(manDir)) {
-      if (path.extname(file) === `.json`) {
-        const fixKey = path.basename(file, `.json`);
-        handleFixtureFile(manKey, fixKey);
+
+if (args.a) {
+  for (const manKey of fs.readdirSync(fixturePath)) {
+    const manDir = path.join(fixturePath, manKey);
+
+    // files in manufacturer directory
+    if (fs.statSync(manDir).isDirectory()) {
+      for (const file of fs.readdirSync(manDir)) {
+        if (path.extname(file) === `.json`) {
+          const fixKey = path.basename(file, `.json`);
+          handleFixtureFile(manKey, fixKey);
+        }
       }
     }
   }
+  checkManufacturers();
+}
+else {
+  // sanitize given path
+  fixturePaths = fixturePaths.map(relativePath => path.resolve(relativePath));
+  for (const fixPath of fixturePaths) {
+    if (path.extname(fixPath) !== `.json`) {
+      // todo: only produce this warning this at a higher verbosity level
```

Comment:
```suggestion
      // TODO: only produce this warning at a higher verbosity level
```

Reply:
Could you please make "todo" all uppercase as I suggested? That's how we spell it elsewhere and my editor is configured to highlight it that way :wink:

Reply:
Sorry, I didn't notice that. We should enforce that via ESLint.